Tests/ApiTests/ApiTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ApiTests\ApiTest;
+
+use App\Database\DatabaseConnection;
+use App\Database\QueryBuilder\MySqlQueryBuilder;
+use App\Repositories\WordRepository;
+use App\Services\FileService;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\TestCase;
+
+class ApiTest extends TestCase
+{
+    private static \PDO $connection;
+    private static WordRepository $wordRepository;
```

Comment:
Why these are static?

Reply:
Update: now I see that you are using setUpBeforeClass, so that is the reason right?

Reply:
Yea